README.md

```diff
@@ -4,7 +4,7 @@
 
 ![Platform](https://img.shields.io/badge/Platform-Linux%20%7C%20macOS%20%7C%20Windows-informational)
 [![Python](https://img.shields.io/badge/Python-%203.8%20%7C%203.9%20%7C%203.10-informational)](https://www.python.org/)
-[![License](https://img.shields.io/badge/License-MIT-green)
+![License](https://img.shields.io/badge/License-MIT-green)

```

Comment:
Good catch, How about we add a hyperlink to "./License/" there? 
Your call, It's fine otherwise too.

Reply:
Sure ! I will do it.